editor/src/components/canvas/canvas-strategies/strategies/reparent-helpers/reparent-property-changes.ts

```diff
@@ -215,27 +221,92 @@ export function getStaticReparentPropertyChanges(
 }
 
 export function positionElementToCoordinatesCommands(
-  elementPath: ElementPath,
+  elementToReparent: ElementPathSnapshots,
+  oldAllElementProps: AllElementProps,
+  metadata: MetadataSnapshots,
   desiredTopLeft: CanvasPoint,
+  pathLookup: OldPathToNewPathMapping,
 ): CanvasCommand[] {
-  return [
-    ...pruneFlexPropsCommands(flexChildProps, elementPath),
+  const basicCommands = [
+    ...pruneFlexPropsCommands(flexChildProps, elementToReparent.newPath),
     setCssLengthProperty(
       'always',
-      elementPath,
+      elementToReparent.newPath,
       PP.create('style', 'top'),
       { type: 'EXPLICIT_CSS_NUMBER', value: cssNumber(desiredTopLeft.y, null) },
       null,
     ),
     setCssLengthProperty(
       'always',
-      elementPath,
+      elementToReparent.newPath,
       PP.create('style', 'left'),
       { type: 'EXPLICIT_CSS_NUMBER', value: cssNumber(desiredTopLeft.x, null) },
       null,
     ),
-    setProperty('always', elementPath, PP.create('style', 'position'), 'absolute'),
+    setProperty('always', elementToReparent.newPath, PP.create('style', 'position'), 'absolute'),
   ]
+
+  const elementIsFragmentLike = treatElementAsFragmentLike(
+    metadata.originalTargetMetadata,
+    oldAllElementProps,
+    metadata.originalPathTrees,
+    elementToReparent.oldPath,
+  )
+
+  if (!elementIsFragmentLike) {
+    return basicCommands
+  }
+
+  const children = replaceFragmentLikePathsWithTheirChildrenRecursive(
```

Comment:
I think you could remove the creation of basicCommands from this function, as well as you manually checking `elementIsFragmentLike`, because `replaceFragmentLikePathsWithTheirChildrenRecursive` will just give you `[originalTarget]` if it was not a fragment-like, and you can map over that single-length array and return that!

Reply:
[0f76da3](https://github.com/concrete-utopia/utopia/pull/3890/commits/0f76da3a6a5188a7f08856845393ba804c6d4786) refactors this code, using less cryptic variable names